File: c/Colleen.c
#include <stdio.h>
/*
	Comment outside of main.
*/
void dummy(){return;}
int main()
{
	/*
		Comment inside main.
	*/
	dummy();
	char str[]="#include <stdio.h>%1$c/*%1$c	Comment outside of main.%1$c*/%1$cvoid dummy(){return;}%1$cint main()%1$c{%1$c	/*%1$c		Comment inside main.%1$c	*/%1$c	dummy();%1$c	char str[]=%2$c%3$s%2$c;%1$c	printf(str, 10, 34, str);%1$c}%1$c";
	printf(str, 10, 34, str);
}

File: python/Colleen.py
#!/usr/bin/python3

"""
    Comment outside of main.
"""
def dummy():
    return
def main():
    """
        Comment inside main.
    """
    dummy()
    mystr = "#!/usr/bin/python3{0}{0}{1}{1}{1}{0}    Comment outside of main.{0}{1}{1}{1}{0}def dummy():{0}    return{0}def main():{0}    {1}{1}{1}{0}        Comment inside main.{0}    {1}{1}{1}{0}    dummy(){0}    mystr = {1}{2}{1}{0}    print(mystr.format(chr(10), chr(34), mystr), end=''){0}{0}if __name__ == {1}__main__{1}:{0}    main(){0}"
    print(mystr.format(chr(10), chr(34), mystr), end='')

if __name__ == "__main__":
    main()
